ggml-cuda/unary.cu

```diff
@@ -92,6 +92,15 @@ static __global__ void sqr_f32(const float * x, float * dst, const int k) {
     dst[i] = x[i] * x[i];
 }
 
+static __global__ void sqrt_f32(const float * x, float * dst, const int k) {
+    const int i = blockDim.x*blockIdx.x + threadIdx.x;
+
+    if (i >= k) {
+        return;
+    }
+    dst[i] = sqrt(x[i]);
```

Comment:
```suggestion
    dst[i] = sqrtf(x[i]);
```

`sqrt` is FP64 square root, `sqrtf` is FP32 square root. And because all NVIDIA GPUs other than H100/A100/V100/... have crippled FP64 performance even a single FP64 instruction can make a large performance difference.